runtime/mount/python_lib/lplots/h5/h5ad/process_message.py

```diff
@@ -15,14 +16,19 @@
 
 ad = auto_install.ad
 
+alignment_is_running = False
+
 
 async def process_h5ad_request(
     msg: dict[str, Any],
     widget_session_key: str,
     adata: ad.AnnData,
     obj_id: str,
+    on_progress: Callable[[object], Awaitable[None]]
```

Comment:
I would just call this send... we will use for other things in here too at some point I assume.